code/datums/gamemode/role/role.dm

```diff
@@ -279,11 +282,33 @@
 
 	to_chat(world, text)
```

Comment:
Probably not meant to stay there

Reply:
I think that the entire proc will have to be reworked at some point.